README.md

```diff
@@ -35,8 +35,10 @@ The ingress host names are as follows, where `<base_hostname>` is your deploymen
 * https://tsa.`<base_hostname>`
 * https://tuf.`<base_hostname>`
 
-## Prerequisites
+## Requirements
 
+* Ansible 2.16.0 or greater
+* Python 3.12.4 or greater
```

Comment:
RHEL 9 has 3.9 and I think there's no RHEL version with a newer Python version. I would suggest requiring >= 3.9 (TBH we don't ship any Python code so we could go even below that... but I guess 3.9 is a reasonable requirement).